README.md

```diff
@@ -7,24 +7,40 @@ For The Win ! is a scoring software for dance competitions, and results archival
 Build & Install
 ---------------
 
-### Dependencies
+### Ocaml Installation
 
-To build Ftw, you will need a working OCaml and Opam installation. To install
+To build Ftw backend, you will need a working OCaml and Opam installation. To install
 OCaml and Opam, you can follow the following instructions:
 
 - For Windows : https://ocaml.org/install#windows
 - For Linux, MacOS, and BSD: https://ocaml.org/install#linux_mac_bsd
 
-Once you have a working OCaml and Opam installation, you'll need to install
-the OCaml dependencies, which can be done using the following command:
+It is recommanded to use ocaml 5.0.0 (see issue #10).
 
-```sh
-opam install . --deps-only
-cd src/frontend && npm install
+Once you have a working ocaml environment, 
+run the following commands to create a switch and activate it.
+```
+opam switch create scrat_ftw_switch 5.0.0
+eval $(opam env)
 ```
 
-You also need to install the various npm dependencies, you can do so with
+### NPM Installation
+
+To build Ftw frontend, you will need a working NPM installation.
+
+To validate that NPM is correctly configured, you can run the following command and check that there are no errors.
+```
+npm --version
+```
+
+### Installation
+
+Once you have a working OCaml and NPM installations, you'll need to install
+the OCaml dependencies, as well as the project's npm depencies.
+It can be done using the following command:
+
 ```sh
+opam install . --deps-only
 cd src/frontend && npm install
```

Comment:
we could replace these two lines with `make configure`, which should do the work